### ISSUE TITLE
Supplemental workbook validation list updates

### ISSUE THREAD

**Issue by paul121** [mstenta/farm_pcsc#30]
"No significant changes, just validation list updates & verification check"

**Comment by mstenta** [comment]
I wish they provided diffs for the validation list changes. Adding those lists was very tedious to begin with, and it seems that the only way to update them is to essentially go through the same process again (thereby making our own diffs).

One example I've already come across (while moving "Field commodity volume unit" from "Field summary" to "Farm commodity volume unit" in "Farm summary"), which illustrates changes that would be easy to miss:

```diff
+          'Bunches',
           'Bushels',
           'Carcass weight pounds',
+          'Cartons',
+          'Eggs',
+          'Flats',
           'Gallons',
           'Head',
           'Linear feet',
-          'Lightweight pounds',
+          'Liveweight pounds',
+          'Pints',
           'Pounds',
           'Tons',
+          'Trees',
           'Other (specify)',
```

It's mostly additions, but one appears to be a typo correction. This would be easy to miss unless we're re-entering all these lists again from scratch.

Part of me wonders if we should just leave the lists as-is and rely on the new "Validation Summary" sheet to catch any potential issues - which we can then fix as they are discovered. That would be less work upfront, but might mean more work fixing things later. I'm leaning towards re-copying all lists from scratch just to ensure we're matching (and crossing fingers that USDA doesn't release a new version with more undocumented changes).

**Comment by mstenta** [comment]
Did a quick pass through validation lists. Documenting the changes that I found (and fixed) here:

**Producer enrollment**

(none)

**Field enrollment**

- Baseline yield unit (6 additions)
- Field irrigated (1 capitalization change)
- Field tillage (1 whitespace change)
- Practice extent unit (1 addition)

**Farm summary**

(see previous comment)

**Field summary**

- Other field measurement (1 option removed)

I have not checked the "Commodity type" list yet. The v2 workbook does not appear to include it, but seems to have a link to it, which is not working for me (says "Invalid range" when I click it in LibreOffice Calc). A list is also provided in the data dictionary PDF, but it's more difficult to copy from there. So this is still TBD.

Note that also I did *not* check the State/County lists because they are gigantic, and I can't imagine they have changed in any meaningful way that would affect us. (Famous last words! :laughing:)
